haiku-driver.groovy

```diff
@@ -92,25 +164,27 @@ def setSpeed(fanspeed){
             sendFanSpeedCommand(7)
             break
     }
-
-    sendEvent(name: "speed", value: "${fanspeed}", isStateChange: true)
 }
 
 def sendFanPowerCommand(String command) {
     sendCommand("FAN", "PWR", command)
 }
 
+def refreshFanSpeed() {
+    sendCommand("FAN", "SPD", "GET;ACTUAL")
+}
+
 def sendFanSpeedCommand(int level) {
     sendCommand("FAN", "SPD", "SET;${level}")
 }
 
 def sendCommand(String haikuSubDevice, String haikuFunction, String command) {
-        def haikuCommand = generateCommand(settings.deviceName, haikuSubDevice, haikuFunction, command)
+        def haikuCommand = generateCommand(haikuSubDevice, haikuFunction, command)
         sendUDPRequest(settings.deviceIp, "31415", haikuCommand)
 }
 
-static def generateCommand(haikuLocation, haikuSubDevice, haikuFunction, command) {
-    return "<${haikuLocation};${haikuSubDevice};${haikuFunction};${command}>"
+static def generateCommand(haikuSubDevice, haikuFunction, command) {
+    return "<ALL;${haikuSubDevice};${haikuFunction};${command}>"
```

Comment:
What's the reason for removing the location?

Reply:
For me, it was hard to know exactly what the fans considered their names, so getting them right in the DeviceName field was difficult. I expect others may have that issue.

Since we're using UDP unicast, not multicast, there's no need for the name to be specified.

Reply:
Makes sense - might add this back later if `InterfaceUtils` ends up allowing device discovery again.